docs/apis/webhooks/sharepoint-webhooks-using-azd-template.md

```diff
@@ -0,0 +1,179 @@
+- An Azure subscription that trusts the same Microsoft Entra ID directory as the SharePoint tenant
+
+## Permissions required to provision the resources in Azure
+
+The account running `azd` must have at least the following roles to successfully provision the resources:
+
+- Azure role [`Contributor`](/azure/role-based-access-control/built-in-roles/privileged#contributor): To create all the resources needed
+- Azure role [`Role Based Access Control Administrator`](/azure/role-based-access-control/built-in-roles/privileged#role-based-access-control-administrator): To assign roles (to access the storage account and Application Insights) to the managed identity of the function app
+
+## Deploy the function app in Azure
+
+1. Run `azd init` from an empty local (root) folder:
+
+    ```shell
+    azd init --template azd-functions-sharepoint-webhooks
+    ```
+
+    Supply an environment name, such as `spofuncs-quickstart` when prompted. In `azd`, the environment is used to maintain a unique deployment context for your app.
+
+1. Open the file `infra/main.parameters.json`, and set the variables `TenantPrefix` and `SiteRelativePath` to match your SharePoint tenant.
+
+   Review the article on [Manage environment variables](/azure/developer/azure-developer-cli/manage-environment-variables) to manage the azd's environment variables.
+
+1. Finally, run the command `azd up` to build the app, provision the resources in Azure and deploy the app package.
+
+## Grant the function app access to SharePoint Online
+
+The authentication to SharePoint is done using `DefaultAzureCredential`, so the credential used depends if the function app runs locally, or in Azure.  
+
+If you never heard about `DefaultAzureCredential`, you should familiarize yourself with its concept by reading [this article](https://aka.ms/azsdk/js/identity/credential-chains#use-defaultazurecredential-for-flexibility).
+
+### Using its managed identity
+
+`DefaultAzureCredential` will use a managed identity to authenticate to SharePoint. This may be the existing, system-assigned managed identity of the function app service or a user-assigned managed identity.  
+
+This tutorial will assume that the system-assigned managed identity is used.
+
+#### Grant the SharePoint API permission Sites.Selected to the managed identity
+
+Navigate to your function app in [the Azure portal](https://portal.azure.com/#blade/HubsExtension/BrowseResourceBlade/resourceType/Microsoft.Web%2Fsites/kind/functionapp) > click `Identity` and note the `Object (principal) ID` of the system-assigned managed identity.  
+
+In this tutorial, it is `d3e8dc41-94f2-4b0f-82ff-ed03c363f0f8`.  
+
+Then, use one of the scripts below to grant this identity the app-only permission `Sites.Selected` on the SharePoint API:
+
+> [!IMPORTANT]
+> The scripts below require at least the delegated permission [`AppRoleAssignment.ReadWrite.All`](/graph/permissions-reference#approleassignmentreadwriteall) (requires admin consent)
```

Comment:
this line & the ones above it that reference GUIDs & scopes should be bold formatting, not inline code

Reply:
Noticed this present in other places in the doc that should also be addressed